packages/matter-node.js-examples/README.md

```diff
@@ -12,6 +12,43 @@ To better show how to use the library we provide some reference implementations
+* `--log-level=...` or environment variable `MATTER_LOG_LEVEL` or in code `environment.vars.set('log.level', Level.DEBUG)`. Allowed values are: Level.FATAL ("fatal"), Level.ERROR ("error"), Level.WARN ("warn"), Level.NOTICE ("notice"), Level.INFO ("info"), Level.DEBUG ("debug")
+* `--log-format=...` or environment variable `MATTER_LOG_FORMAT` or `environment.vars.set('log.format', Format.PLAIN)`. Allowed values are: Format.PLAIN ("plain"), Format.HTML ("html"), Format.ANSI ("ansi")
+* `--storage-path=...` or environment variable `MATTER_STORAGE_PATH` or `environment.vars.set('storage.path', "...")` allows to set the storage location. By default, it will be stored in the users home directory in `.matter/...`. If specified, the given path will be used relative to the current working directory.
+* `--storage-clear` or environment variable `MATTER_STORAGE_CLEAR=1` or `environment.vars.set('storage.clear', true)` allows to define if the storage is reset on startup.
+* `--mdns-networkinterface=...` or environment variable `MATTER_MDNS_NETWORKINTERFACE` or `environment.vars.set('mdns.networkInterface', "...")` allows to limit the DNS announcements and scanning to one network interface. By default, all available interfaces are used.
+Additionally, all command line parameters now require to start with two dashes!
+
+* **BridgedDevicesNode**: This example shows how to build a Matter-Bridge to offer multiple OnOff lights and sockets as a bridge. It can be configured via command line and allow to add shell commands to be executed for on and off commands for the numbered devices. More details see below.
+* **ComposedDeviceNode**: This example shows how to build a "simple" composed device where multiple OnOff lights and sockets are combined as one composed device. The devices are all added on root level, so no bridge is used. It can be configured via command line and allow to add shell commands to be executed for on and off commands for the numbered devices. More details see below.
+* **DeviceNode**: This example shows how to build a simple minimalistic DeviceNode with just one socket or light endpoint. The shell commands to be executed by on/off commands can be configured via CLI.
+* **DeviceNodeFull**: This example shows how to build a simple Device node with just one socket or light endpoint. The shell commands to be executed by on/off commands can be configured via CLI. Additionally, this example also shows how to enable BLE for a device node and tweaks the announcement that only BLE is announced in the beginning. For this is also includes a dummy WifiNetworkCommissioning and a dummy ThreadNetworkCommissioning implementation that simulates Wi-Fi/Thread logic for the commissioner. Furthermore, it also enhances the GeneralDiagnostics and OnOff cluster to implement some commands with own logic. It also implements the callbacks where implementers can get information on commissioning changes and session/connection changes to better know the status of the node.
```

Comment:
* **DeviceNodeFull**: This example shows how to build a simple Device node with just one socket or light endpoint. The shell commands to be executed by on/off commands can be configured via CLI. Additionally, this example also shows how to enable BLE for a device node and tweaks the announcement so that BLE is only announced in the beginning. This example also includes a dummy WifiNetworkCommissioning and a dummy ThreadNetworkCommissioning implementation that simulates Wi-Fi/Thread logic for the commissioner. Furthermore, it also enhances the GeneralDiagnostics and OnOff cluster to implement some commands with its own logic. The example also implements the callbacks where implementers can get information on commissioning changes and session/connection changes to better know the status of the node.